src/webvr-polyfill.js

```diff
@@ -107,7 +107,7 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
   navigator.getVRDisplays = this.getVRDisplays.bind(this);
 
   // Polyfill native VRDisplay.getFrameData
-  if (this.nativeWebVRAvailable && window.VRFrameData) {
+  if (this.nativeWebVRAvailable && this.isCardboardCompatible() && window.VRFrameData) {
```

Comment:
Does this have any side effects when the condition is legitimate? Should the fix go in the implementation of `isCardboardCompatible` so it does not give false positives / negatives?